### PR TITLE
DOCS-1565: Add class filter to transform camera

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/components/camera/transform.md [hunk 1]
```diff
@@ -140,6 +142,8 @@ Classifications overlay text from the `GetClassifications` method of the [vision
 
 - `classifier_name`: The name of the classifier in the [vision service](/ml/vision/).
 - `confidence_threshold`: The threshold above which to display classifications.
+- `max_classifications`: The maximum number of classifications to display on the camera stream at any given time.
+- `valid_labels`: An array of labels that you to see detections for on the camera stream.
```
Comment (sguequierre) [inline docs/components/camera/transform.md:146]: Do we know for sure that these are all required? Would be good to point out if not.

**Review by sguequierre — APPROVED** [review]
LGTM % one comment

File: docs/components/camera/transform.md (hunk 1 above)
Comment (npentrel) [reply docs/components/camera/transform.md:146]: I don't think they are but will check

File: docs/components/camera/transform.md [hunk 2]
```diff
@@ -244,6 +249,7 @@ The Detections transform takes the input image and overlays the detections from
 
 - `detector_name`: The name of the detector configured in the [vision service](/ml/vision/).
 - `confidence_threshold`: Specify to only display detections above the specified threshold (decimal between 0 and 1).
+- `valid_labels`: An array of labels that you to see detections for on the camera stream.
```
Comment (npentrel) [inline docs/components/camera/transform.md:252]:
```suggestion
- `valid_labels`: _Optional_. An array of labels that you to see detections for on the camera stream.
```

File: docs/components/camera/transform.md (hunk 1 above)
Comment (kharijarrett) [reply docs/components/camera/transform.md]: These are both optional. They have "under the hood" defaults that may or may not be worth sharing. If you don't put anything for max_classifications, it'll just put one classification (which is what we've always had and what most expect). If you don't put anything for valid_labels, it'll display all the labels your classifier returns (again, expected and always had).

**Review by kharijarrett — APPROVED** [review]
LGTM! I left a comment about things being optional

File: docs/components/camera/transform.md [hunk 3]
```diff
@@ -140,6 +142,8 @@ Classifications overlay text from the `GetClassifications` method of the [vision
 
 - `classifier_name`: The name of the classifier in the [vision service](/ml/vision/).
 - `confidence_threshold`: The threshold above which to display classifications.
+- `max_classifications`: _Optional_. The maximum number of classifications to display on the camera stream at any given time.
```
Comment (npentrel) [inline docs/components/camera/transform.md:145]:
```suggestion
- `max_classifications`: _Optional_. The maximum number of classifications to display on the camera stream at any given time. Default: `1`.
```

File: docs/components/camera/transform.md [hunk 4]
```diff
@@ -140,6 +142,8 @@ Classifications overlay text from the `GetClassifications` method of the [vision
...
+- `valid_labels`: _Optional_. An array of labels that you to see detections for on the camera stream.
```
Comment (npentrel) [inline docs/components/camera/transform.md:146]:
```suggestion
- `valid_labels`: _Optional_. An array of labels that you to see detections for on the camera stream. If not specified, all labels from the classifier are used.
```

File: docs/components/camera/transform.md [hunk 5]
```diff
@@ -244,6 +249,7 @@ The Detections transform takes the input image and overlays the detections from
 
 - `detector_name`: The name of the detector configured in the [vision service](/ml/vision/).
 - `confidence_threshold`: Specify to only display detections above the specified threshold (decimal between 0 and 1).
+- `valid_labels`: _Optional_. An array of labels that you to see detections for on the camera stream.
```
Comment (npentrel) [inline docs/components/camera/transform.md:252]:
```suggestion
- `valid_labels`: _Optional_. An array of labels that you to see detections for on the camera stream. If not specified, all labels from the classifier are used.
```

**Pull request merged**